tests/csrf_tests/test_context_processor.py

```diff
@@ -0,0 +1,15 @@
+import json
```

Comment:
I would have been satisfied with the other test, but since you've written this one I don't have any problem with adding it.